UPGRADING.md

```diff
@@ -173,6 +177,8 @@ The return type of the interface method `TxConfig.SignModeHandler()` has been ch
 The `sdk.Msg` interface has been updated to not require the implementation of the `ValidateBasic` method.
 It is now recommended to validate message directly in the message server. When the validation is performed in the message server, the `ValidateBasic` method on a message is no longer required and can be removed.
 
+Messages no longer need to implement the `LegacyMsg` interface and implementations of `GetSignBytes` can be deleted.  Because of this change, global legacy Amino codec definitions and their registration in `init()` can safely be removed as well.  
```

Comment:
We need to explain how to add the amino annotations

Reply:
there is an issue to write docs on this, i start but didnt complete. Ill pick it up next week along with other doc changes for eden